src/service/rooms.js

```diff
@@ -94,12 +48,11 @@ const createHandler = async (req, res) => {
       madeat: Date.now(),
       maxPartLength: maxPartLength,
       settlementTotal: 0,
-      isOver: false,
     });
-    await room.save();
 
     // 방의 ObjectID를 방 생성 요청을 한 사용자의 room 배열에 추가
-    user.room.push(room._id);
+    user.ongoingRoom.push(room._id);
```

Comment:
"populate"해도 `objectId`만 push하고도 save가 되는가 보군요!!

Reply:
오 네네! Query result에 populate 연산을 적용해도 여전히 원본 document에 대한 reference가 유지된다고 해요.

> The documents returned from [query population](https://mongoosejs.com/docs/api.html#query_Query-populate) become fully functional, removeable, saveable documents unless the [lean](https://mongoosejs.com/docs/api.html#query_Query-lean) option is specified. Do not confuse them with [sub docs](https://mongoosejs.com/docs/subdocs.html). Take caution when calling its remove method because you'll be removing it from the database, not just the array. - [Mongoose v6.7.1 Populate 문서](https://mongoosejs.com/docs/populate.html#refs-to-children)